spec/requests/disability_compensation_form_request_spec.rb

```diff
@@ -246,6 +246,32 @@ def test_error(cassette_path, status, headers)
           expect(form.dig('form526', 'form526', 'toxicExposure')).not_to eq(nil)
         end
 
+        it 'sets claims_api as submit_endpoint when includes_toxic_exposure query param value is true' do
+          query_param = '?includes_toxic_exposure=true'
+          post("/v0/disability_compensation_form/submit_all_claim#{query_param}", params: all_claims_form, headers:)
+          expect(response).to have_http_status(:ok)
+          expect(response).to match_response_schema('submit_disability_form')
+          submission = Form526Submission.last
```

Comment:
Using `first` and `last` to test a side effect of an action always makes me nervous because it's not deterministic. Is there any way around this you can think of?

Reply:
If the `/v0/disability_compensation_form/submit_all_claim` endpoint is designed to create a single `Form526Submission` record, then using `Form526Submission.last` to retrieve that record for testing should be reliable. But I did add to each test an expectation that there is only a single `Form526Submission` record.

Reply:
Yeah that's one way to get around it which is probably fine for now. 

Outside of this example: my concern with using first and last in general is there could be side effects elsewhere in the code creating records you might not know about or other data created by a before block in the tests or anything like that. Or data that gets seeded for running every test. And then maybe the endpoint doesn't work as expected and because there is a record that already existed that would get returned from `Form526Submission.last` instead.  It's probably unlikely but something to consider when writing tests. What I'm basically saying is assuming the latest record in a table was one you created without any other identifier could lead to problems if that makes sense.